src/main/java/janet/Ui.java

```diff
@@ -0,0 +1,97 @@
+package janet;
+
+/**
+ * Encapsulates methods needed for Janet to interact with the user through printing
+ * messages to the console.
+ */
+public class Ui {
+
+    public static final String EXPECTED_SCHEDULE_QUERY = "The date for which you wish to view tasks cannot be empty.";
+    public static final String EXPECTED_DATE_GOT_OTHER =
+            "The date for which you wish to view tasks should be of the format YYYY-MM-DD.";
+    static final String EXPECTED_SEARCH_QUERY = "The search query of a find operation cannot be empty.";
+    static final String INTRO_STRING = "Hey there! I'm Good Janet. How many I help you today?";
+    static final String EXPECTED_DONE_INDEX_GOT_NONE =
+            "You need to indicate which task number should be marked as done.";
+    static final String EXPECTED_DONE_INDEX_GOT_OTHER =
+            "The task to be marked as done should be indicated its list index.";
+    static final String EXPECTED_TO_DO_DESCRIPTION = "The description of a todo cannot be empty.";
+    static final String EXPECTED_DEADLINE_DESCRIPTION = "The description of a deadline cannot be empty.";
+    static final String EXPECTED_DEADLINE_BY = "The done-by date of a deadline cannot be empty.";
+    static final String EXPECTED_EVENT_DESCRIPTION = "The description of an event cannot be empty.";
+    static final String EXPECTED_EVENT_AT = "The timing of an event cannot be empty.";
+    static final String EXPECTED_DELETED_INDEX_GOT_NONE = "You need to indicate which task number should be deleted.";
+    static final String EXPECTED_DELETED_INDEX_GOT_OTHER = "The task to be deleted should be indicated its list index.";
+    static final String UNRECOGNISED_OPERATION = "Sorry, I do not understand this command.";
+    static final String INDEX_OUT_OF_BOUNDS_STRING = "Sorry, there is no task with the index number ";
+
+    /**
+     * Returns a string representation of the tasks in the list.
+     *
+     * @param taskList Janet's list of tasks
+     * @return String representation of list
+     */
+    public static String taskListString(TaskList taskList) {
```

Comment:
Perhaps you could name this method as printTaskListString so that it sounds more like a verb?

Reply:
Ahh, good catch. Thank you!